### DISCUSSION TITLE
When I try to sign into the Microsoft MyApps portal to authorize, I get this error.

### DISCUSSION THREAD

**Discussion by aidoit** [abraunegg/onedrive#2346] [Q&A] [answered]
access denied
The User ID you entered was not recognized. If this is a new account, it may take several hours to access the service.
Important: This page does not yet support signing in with a Microsoft account. If you already have an account assigned to your organization, try logging in again with that account. If you don't have this account, please ask your administrator for a new work or school account and try again.

How do i fix it？

**Comment by abraunegg** [comment, marked as answer]
@aidoit 
> How do i fix it？

The User ID you entered was not recognized. If this is a new account, it may take several hours to access the service.

Suggest that you reach out to Microsoft here for assistance as this is nothing I can do to assist you.
